index.html

```diff
@@ -60,18 +60,18 @@ <h2 id="novedades">Novedades</h2>
                     Si estás interesado en vender tu auto, nosotros lo evaluamos y te damos una cotización.
                     Podés utilizar este valor para cambiarlo por otro auto de nuestro catálogo o simplemente venderlo.
                 </p>
-                <button>Cotizá tu auto</button>
+                <button onclick="location.href='cotiza.html'">Cotizá tu auto</button>
             </div>
             <div class="consulta">
                 <img src="assets/Consulta mecanica.jpg" alt="Consulta técnica">
                 <p>¿Querés sacarte una duda sobre un auto, forma de pago u otra cosa?</p>
-                <button>Hacenos tu consulta</button>
+                <button onclick="location.href='contacto.html'">Hacenos tu consulta</button>
```

Comment:
```suggestion
                <a class="btn" href="contacto.html'">Hacenos tu consulta</a>
```
Después se puede dar estilo de boton al enlace, evitemos el uso de eventos dentro del HTML embebido, para eso vamos a usar JavaScript más adelante